go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -291,6 +294,19 @@ func (be *BuiltinBackupEngine) executeIncrementalBackup(ctx context.Context, par
 	if err != nil {
 		return false, vterrors.Wrapf(err, "cannot get binary logs in incremental backup")
 	}
+
+	// @@gtid_purged
```

Comment:
Is this comment line needed? Doesn't seem to add anything.

Reply:
Removed.